docs/core/development/policy/code-dev.rst

```diff
@@ -3,7 +3,7 @@
 :LastChangedRevision: $LastChangedRevision$
 :LastChangedBy: $LastChangedBy$
 
-Working from Twisted's Subversion repository
+Working from Twisted's code repository
```

Comment:
I feel like this should say Git rather than code, "code repository" sounds a bit eh to me.